bin/start.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+set -e
+
+if [ -f "${DOCKER_CERT_PATH}/ca.pem" ]
```

Comment:
The specific case here is wrapping ContainerPilot, which means we lose PID 1 and we should probably always try to avoid, but there's a general question of when to wrap your app vs. use a `preStart` and what the tradeoffs might be.

Both of those points might deserve coverage in the docs @geek has been working on.